docs/concepts/concept-pt1-bitmap-index.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Part 1 - Bitmap indexes
+layout: default
+parent: Bitmap index overview
+grand_parent: Concepts
+nav_order: 1
+---
+
+# Part 1 - bitmap indexes
+{: .no_toc }
+
+This is the first in a series of articles that explains how FeatureBase uses bitmaps, bit-slicing and range-encoding to:
```

Comment:
no range encoding. Not commenting on every instance of it, but it will need to be removed

Reply:
removed